WMass/python/plotter/w-helicity-13TeV/wmass_e/wenu_80X.txt

```diff
@@ -3,5 +3,8 @@ HLT_SingleEL : HLT_SingleEl == 1
 onelep : nLepGood == 1 && abs(LepGood1_pdgId)==11
 fiducial : abs(LepGood1_eta)<1.4442 || abs(LepGood1_eta)>1.566
 eleKin : ptElFull(LepGood1_calPt,LepGood1_eta) > 30 && ptElFull(LepGood1_calPt,LepGood1_eta) < 45 && abs(LepGood1_eta)<2.5
+highEtaEE_HLTlowPt: abs(LepGood1_eta) < 2.1 || HLT_BIT_HLT_Ele27_WPTight_Gsf_v == 1 || HLT_BIT_HLT_Ele25_WPTight_Gsf_v == 1
```

Comment:
like this for |eta|<2.1 the HLT is not required. The WENU skim requires HLT_SingleEL, but still we should always require a trigger. Maybe I would try `HLT_BIT_HLT_Ele27_WPTight_Gsf_v == 1 || HLT_BIT_HLT_Ele25_WPTight_Gsf_v == 1` everywhere

Reply:
OK, there is still the requirement of the OR of the SingleElectron